test/functional/test_framework/test_node.py

```diff
@@ -171,7 +172,7 @@ def __getattr__(self, name):
             assert self.rpc_connected and self.rpc is not None, self._node_msg("Error: no RPC connection")
             return getattr(self.rpc, name)
 
-    def start(self, extra_args=None, *, stdout=None, stderr=None, **kwargs):
+    def start(self, extra_args=None, cwd=None, stdout=None, stderr=None, **kwargs):
```

Comment:
```suggestion
    def start(self, extra_args=None, *, cwd=None, stdout=None, stderr=None, **kwargs):
```

Any reason why you removed the star? It should stay there

Reply:
Oops, I misread your original comment.